draft-irtf-cfrg-frost.md

```diff
@@ -261,6 +261,8 @@ We now detail a number of member functions that can be invoked on `G`.
 - DeserializeScalar(buf): Attempts to map a byte array `buf` to a `Scalar` `s`.
   This function can raise a DeserializeError if deserialization fails; see
   {{ciphersuites}} for group-specific input validation steps.
+- IdentifierToScalar(i): Maps an identifier `i` to a Scalar with the same numeric
```

Comment:
Why is this a member function of the group? It seems like a helper function to me. Can we move it to that part of the draft, and then actually specify how it's implemented?

Reply:
It's related to a Scalar, just like `DeserializeScalar` but with a different input. I don't see a way to specify the implementation in a generic manner, it will depend on how `Scalar` is implemented.

Reply:
Sure, but it's a function that one can invoke _with_ group elements, not _on_ a group.

Reply:
Yeah this should be a helper function vs in the Group definitions

Reply:
@conradoplg will you be able to relocate this, or would you like one of us to do it?

Reply:
> @conradoplg will you be able to relocate this, or would you like one of us to do it?

I can do it, but I'm having trouble working out the best way to do it. Some possible approaches:

- Move to "Helper functions": these are all in pseudo-code, but IdentifierToScalar can't be described as pseudo-code (since Scalar is implementation-dependent)
- Keep in this section, but add a new bullet list of helper functions
- Rename to "IntegerToScalar" and keep it in this list (and the Identifier -> integer conversion will be implied)

Reply:
Hmm, interesting point! How about something like the last option, but maybe spelled like this:

```
We denote Scalar(x) as the conversion of integer input `x` to the corresponding Scalar value with the same numeric value. For example, Scalar(1) yields a Scalar representing the value 1.
```

We already use the notation `Scalar(0)` and `Scalar(1)` in other places, so this would just formalize that convention. We could put this in the paragraph at the top of section 3.1.

Thoughts?

Reply:
Ahh neat, I missed that this notation was already used elsewhere! That seems the best approach, I'm make this change shortly

Reply:
Done in b7145dd40e6